main.go

```diff
@@ -17,9 +17,6 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	if fi.Mode()&os.ModeNamedPipe == 0 {
```

Comment:
just removing this won't work, as then this won't compile, as `fi` is unused

Reply:
if we just change line 16 to

```
	_, err := os.Stdin.Stat()
```

then it should work

Reply:
thanks for the catch, I pushed the fix